File: EduSafe.WebSite/src/app/classes/individuals/modelOutputSummary.ts
import { ModelOutputHeaders } from './modelOutputHeaders';
import { ModelOutputEntry } from './modelOutputEntry';

export class ModelOutputSummary {
  OutputTitle: string;
  DropOutCoveragePercentage: number;
  GradSchoolCoveragePercentage: number;
  EarlyHireCoveragePercentage: number;
  ModelOutputHeaders: ModelOutputHeaders;
  ModelOutputEntries: ModelOutputEntry[];
}

File: EduSafe.WebApi/Models/Individuals/ModelOutputSummary.cs
using System.Collections.Generic;

namespace EduSafe.WebApi.Models.Individuals
{
    public class ModelOutputSummary
    {
        public string OutputTitle { get; set; }
        public int DropOutCoveragePercentage { get; set; }
        public int GradSchoolCoveragePercentage { get; set; }
        public int EarlyHireCoveragePercentage { get; set; }
        public ModelOutputHeaders ModelOutputHeaders { get; set; }
        public List<ModelOutputEntry> ModelOutputEntries { get; set; }
    }
}